src/DAL/Events/Event.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace DAL.Events
+{
+	public sealed class Event
+	{
+		public int Id { get; set; }
+		public string Name { get; set; }
+		public string Description { get; set; }
+		public int VenueId { get; set; }
+		public Venue.Venue Venue { get; set; }
```

Comment:
Maybe you can simplify this name with using?

Reply:
It seemed easier to make namespaces plural...